src/components/Header/Header.tsx

```diff
@@ -0,0 +1,24 @@
+import "./Header.css";
+import Girl from "../../assets/girl.svg";
+import Baikal from "../../assets/Baikal.svg";
+import GirlAndBoy from "../../assets/girl_and_boy.svg";
+
```

Comment:
https://create-react-app.dev/docs/importing-a-component/#absolute-imports

Reply:
Thank you! :)